frontend/packages/kubevirt-plugin/integration-tests/tests/utils/mocks.ts

```diff
@@ -237,6 +238,49 @@ export function getVMManifest(
       throw Error('Provision source not Implemented');
   }
 
+  const spec = {
```

Comment:
can you please rename this to `VMISpec` ?

Reply:
sure, done